db/error_handler.cc

```diff
@@ -16,6 +16,22 @@ namespace rocksdb {
 // (i.e from all 4 fields in a tuple to only the BackgroundErrorReason and
 // paranoid_checks). The less specific map serves as a catch all in case we miss
 // a specific error code or subcode.
+std::map<std::tuple<BackgroundErrorReason, bool>, Status::Severity>
+    RetryableErrorMap = {
+        {std::make_tuple(BackgroundErrorReason::kCompaction, true),
```

Comment:
Annotating `true` with `/*retryable*/` can make it clearer. Or add a sentence to explain what it means.

Reply:
I'm actually planning to revert the changes to error_handler.cc. Originally, I combined the new API with some handling of retryable errors, but I think its better to separate it into another PR with a proper design.